apps/emqx_dashboard/i18n/emqx_dashboard_i18n.conf

```
@@ -197,4 +197,14 @@ its own from which a browser should permit loading resources."""
       zh: "多语言支持"
     }
   }
+  bootstrap_user {
+    desc {
+      en: "Initialize users file."
+      zh: "初始化用户文件"
```

```suggestion
      en: "Initialize users file."
      zh: """用于在首次启动 emqx 时，为 Dashboard 添加管理用户，其格式为：
    ```
    username1:password1
    username2:password2
    ```
"""
```